tools/replay_gen.py

```diff
@@ -317,12 +348,22 @@ def parse_args():
                         help='Path to a directory where the replay files are generated.')
     parser.add_argument('-p', '--prefix', dest='prefix', type=str, default='',
                         help='Prefix for the replay file names.')
+    parser.add_argument(
+        '-j',
+        '--out-json',
+        dest='out_json',
+        default=False,
+        action='store_true',
+        help='Dump replay files in JSON format. By default replay files will be formatted as YAML.')
     return parser.parse_args()
 
 
 def main():
     args = parse_args()
 
+    if not has_yaml:
+        args.out_json = True
+
```

Comment:
Probably best not to modify args directly. If inspected later through pdb, for instance, one would normally expect the parse_args to be the parsed output. I'd recommend a local variable. Maybe something like:

```
out_json = args.out_json or not has _yaml
```